day_two_guide/host_level_tasks.adoc

```diff
@@ -0,0 +1,63 @@
+[[day_two_host_level_tasks]]
+= Host-level tasks
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Adding a host to the cluster
+
+For information on adding master or node hosts to a cluster, see the xref:../install_config/adding_hosts_to_existing_cluster.adoc[Adding hosts to an existing cluster] section in the install and configuration guide.
+
+== Master host tasks
+
+[[day-two-guide-deprecating-master]]
+=== Deprecating a master host
+
+include::day_two_guide/topics/deprecating_master.adoc[]
+
+[[day-two-guide-master-backup]]
+=== Creating a master host backup
+
+include::day_two_guide/topics/master_backup.adoc[]
+
```

Comment:
I think it would be nice to add the "restore" section as well (it is included in the original masters_backup.adoc from the internal gitlab)